department-of-veterans-affairs/caseflow
# Allow AMA AttorneyTasks to be cancelled

Connects #12115

### Description
Adds task action to attorney task to allow them to be cancelled and sent back to the assigning judge to be assigned again.

![Screen Shot 2019-10-07 at 11 46 57 AM](https://user-images.githubusercontent.com/45575454/66327148-52313200-e8f8-11e9-8cfb-e062e2c0d660.png)
![Screen Shot 2019-10-07 at 11 47 04 AM](https://user-images.githubusercontent.com/45575454/66327164-58bfa980-e8f8-11e9-930e-886360ad890c.png)
![Screen Shot 2019-10-07 at 11 47 15 AM](https://user-images.githubusercontent.com/45575454/66327168-5a896d00-e8f8-11e9-922d-4714770aa5a0.png)
![Screen Shot 2019-10-07 at 11 47 28 AM](https://user-images.githubusercontent.com/45575454/66327171-5c533080-e8f8-11e9-8943-4be33270e4ad.png)

### Acceptance Criteria
- [ ] Attorneys can cancel AMA `AttorneyTask`s assigned to them

### Testing Plan
1. Sign in as "BVASCASPER1"
2. Go to any assigned task in the user's queue
3. Ensure they have the 'Cancel Task' action
4. Ensure cancelling the task removes it from the user's queue and sends the task back to the judge
